public/js/index.js

```diff
@@ -147,3 +147,9 @@ function selectivelyParse() {
 }
 editor.on('change', (event) => { selectivelyParse(); });
 
+window.addEventListener('load', function() {
+    pageUrl = new URL(window.location.href);
+    if(pageUrl.searchParams.get('url')){
+        document.getElementById('oas_url').value = pageUrl.searchParams.get('url');
+    }
```

Comment:
Is that possible to trigger the validation without relying on the button?

Reply:
@ioggstream sure we just need to call `parseUrl()` after line 153 
and no - load will just wait the page to be loaded

Reply:
i added the `parseUrl()` call

Reply:
I know, I'm a nuisance, could we keep same lint rule (2 spaces) for this snippet?

Reply:
I'll do it :P